lib/Template.js

```diff
@@ -152,12 +152,12 @@ module.exports = class Template {
 			var maxId = bounds[1];
 			if(minId !== 0) source.add("Array(" + minId + ").concat(");
 			source.add("[\n");
-			const modules = {};
+			const modules = new Map();
 			allModules.forEach(module => {
-				modules[module.id] = module;
+				modules.set(module.id, module);
```

Comment:
```js
const modules = new Map(allModules.map(module => [module.id, module]))
```

looks better to me 🤔

Reply:
It creates a closure. I (or someone else 😉 ) will address this in another PR.